src/Albany_Utils.cpp

```diff
@@ -191,31 +191,6 @@ getParameterSizes(const Teuchos::ParameterList parameterParams, int &total_num_p
   }
 }
 
-void
```

Comment:
Is "ReplaceDiagonalEntries" not used?

Reply:
Never. I grepped the library, and it's not. I guess it might have been used in LCM, or one of those packages. Since fixing it was giving me some trouble (not much, just a little hiccup), I figured I would just remove it, and call it a day.